server/tournaments/index.ts

```diff
@@ -607,6 +607,7 @@ export class Tournament extends Rooms.RoomGame {
 		}
 		this.isBracketInvalidated = true;
 		this.room.add(`|tournament|start|${this.players.length}`);
+		this.modlog('TOUR START', null, `The tournament has started with ${this.players.length} users.`);	// Modnote number of players and when a tour starts
```

Comment:
```suggestion
		this.modlog('TOUR START', null, `The tournament has started with ${this.players.length} users.`);
```

This comment is useless.